fishy/engine/semifisher/fishing_event.py

```diff
@@ -172,14 +178,23 @@ def on_invfull():
     logging.info(msg)
     web.send_notification(msg)
 
+    if FishEvent.sound:
+        playsound(helper.manifest_file("sound.mp3"), False)
+
 
 def on_fight():
     msg = "FIGHTING!"
     logging.info(msg)
     web.send_notification(msg)
 
+    if FishEvent.sound:
+        playsound(helper.manifest_file("sound.mp3"), False)
+
 
 def on_dead():
     msg = "Character is dead!"
     logging.info(msg)
     web.send_notification(msg)
+
+    if FishEvent.sound:
+        playsound(helper.manifest_file("sound.mp3"), False)
```

Comment:
as seeing on_fight, on_dead, on_invfull, on_nobait does the exact same thing, u can use just create one function which takes in msg, parameter and use lambda functions in the callbacks_map